test/integration/other/db-stream.js

```diff
@@ -35,6 +38,7 @@ describe('db.stream()', () => {
 
     // TODO this doesn't seem like the correct behaviour - no Error is thrown
     // here.  At some point, this should be fixed elsewhere!
+    // REVIEW: this behaviour has been fixed in Slonik
     it('should close streams given an invalid sql statement', async () => {
```

Comment:
If this behavior has been fixed, should we remove this test?

Reply:
I'd happily remove them all...